functions-builder.toml

```diff
@@ -36,6 +40,10 @@ version = "0.4.0"
     id = "heroku/nodejs-npm-buildpack"
     version = "0.1.2"
 
+  [[order.group]]
+    id = "salesforce/sf-fx-middleware"
+    version = "0.0.4"
```

Comment:
do we want this to be optional like [procfile buildpack](https://github.com/heroku/pack-images/blob/master/builder.toml#L111):

```
optional = true
```

Reply:
option=true... does it mean if detect fails (return false)... the builder will just build the source w/o the middleware buildpack?

Reply:
yes

Reply:
@jkutner @jqian2017  At first, I was thinking of having an entirely new `[[order]]` for the middleware + npm + nodejs buildpacks, but having `optional = true` seems like a good approach.  I image down the road, if we have other middleware buildpacks that are mutually exclusive from other buildpacks, we'd want to separate those into separate `[[order]]` sections?